LICENSE

```diff
@@ -1,6 +1,7 @@
 MIT License
 
-Copyright (c) 2017-2019 Metarhia contributors (full list in AUTHORS file)
+Copyright (c) 2017-2020 Metarhia contributors
+(https://github.com/metarhia/common/blob/master/AUTHORS).
```

Comment:
`AUTHORS` file is published with the package, so I don't see the point in adding the link to it here.
Besides, it's possible for the repository to move somewhere, thus invalidating this link.